src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -34,12 +34,16 @@ public partial class BrotliStream : Stream
         Decoder _decoder;
         Encoder _encoder;
 
-        public BrotliStream(Stream baseStream, CompressionMode mode, bool leaveOpen, int bufferSize, uint windowSize, uint quality) : this(baseStream, mode, leaveOpen, bufferSize)
+        public override int ReadTimeout { get => base.ReadTimeout; set => base.ReadTimeout = value; }
```

Comment:
realization was added , but I'm not sure that I do it in a best way. Check time on every loop iteration looks very slowly for me.